src/results-output/results-doc.ts

```diff
@@ -13,6 +13,7 @@ import * as replHistory from './repl-history';
 import * as docMirror from '../doc-mirror/index'
 import { PrintStackTraceCodelensProvider } from '../providers/codelense';
 import * as replSession from '../nrepl/repl-session';
+import { max } from 'lodash';
```

Comment:
This goes with my suggestion lower in the file.

```suggestion
import { max, takeWhile } from 'lodash';
```